### DISCUSSION TITLE
Validation against Tags - Object Type

### DISCUSSION THREAD

**Discussion by ghost** [netbox-community/netbox#15068] [Q&A]
I'm trying to add custom validation to ensure a tag is assigned an [Object Type](https://docs.netbox.dev/en/stable/models/extras/tag/#object-types) before saving. I've added a required flag to this field, however, even if an object is selected the form returns 'This field must not be empty'. Is there another way to validate against these multi fields without using class validation?

Validations
![image](https://github.com/netbox-community/netbox/assets/140499038/ef6c05e7-1203-4a68-8c8e-63beb6035a96)

Form output:
![image](https://github.com/netbox-community/netbox/assets/140499038/8013cebf-f1f9-40e9-be4f-f7f55c42dccb)

Edit: Added context / changed title

**Comment by candlerb** [comment]
By "object list", do you mean you have created a [Custom Field](https://docs.netbox.dev/en/stable/customization/custom-fields/) of type [multi-object](https://docs.netbox.dev/en/stable/customization/custom-fields/#custom-object-fields)?

> even when I select an item from the list it still reports back as empty

What do you mean by "reports back as empty?"

**Reply by ghost** [reply]
Sorry - [Object Type](https://docs.netbox.dev/en/stable/models/extras/tag/#object-types) not list.

I added a 'required' validation against the Object Type assuming that Netbox would check if an object had been selected, however, even if I select an object before I submit it will error stating that the field is empty.
